### PR TITLE
Test(Qtm-322) Test migration - Enzyme to text-library: Accordion, Alert and Avatar

### PR DESCRIPTION
**Pull request by matheusrpp**
## Description
https://jirasoftware.catho.com.br/browse/QTM-322

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Code review

### CODE REVIEW

File: components/Accordion/Accordion.jsx [hunk 1]
```diff
@@ -55,8 +55,6 @@ const AccordionHeader = styled.button.attrs({ type: 'button' })`
   display: inline-flex;
   justify-content: space-between;
   width: 100%;
-
-  ${({ as }) => (as === 'button' ? `cursor: pointer;` : null)}
```
Comment (toncabral88) [inline components/Accordion/Accordion.jsx:59]: Why did you remove this?

File: components/Accordion/__snapshots__/Accordion.unit.test.jsx.snap [hunk 2]
```diff
@@ -95,111 +95,31 @@ exports[`Accordion component Should match the snapshot 1`] = `
...
-    className="c1 accordion-item"
+    class="c1 accordion-item"
     color="#191919"
   >
     <button
       aria-controls="Institucional-0-content"
-      className="c2"
+      class="c2"
       id="Institucional-0-header"
-      onClick={[Function]}
```
Comment (toncabral88) [inline components/Accordion/__snapshots__/Accordion.unit.test.jsx.snap:108]: Don't snaps show anymore theme and events? I don't know if this is a problem. What do you think?

File: package.json [hunk 3]
```diff
@@ -76,6 +76,8 @@
     "@storybook/router": "^6.4.9",
     "@storybook/storybook-deployer": "^2.8.10",
     "@storybook/theming": "^6.4.9",
+    "@testing-library/jest-dom": "^5.16.1",
```
Comment (toncabral88) [inline package.json:79]: What this plugin do?

**Review by toncabral88 — APPROVED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Code review

Another question. Do we stop using react-test-renderer too?

**Review by rapahaeru — APPROVED** [review]
## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Code review

Antes
image

Depois da migração
image

Repare que a cobertura do Accordion e Alert diferenciam. Acham relevante monitorar isso?

File: components/Accordion/Accordion.jsx (hunk 1 above)
Comment (rapahaeru) [reply components/Accordion/Accordion.jsx:59]: i want to know too rs

Comment (matheusrpp) [reply components/Accordion/Accordion.jsx:59]: Because when I went to see why this guy was not being covered in the unit tests, I notice that this prop is always undefined, and in line 54, there is a `cursor: pointer` already.
